packages/functional/pages/checkout-page.ts

```diff
@@ -0,0 +1,20 @@
+export const GUEST_CUSTOMER_EMAIL = '#email';
+export const CONTINUE_AS_GUEST = '[data-test="customer-continue-as-guest-button"]';
+export const FIRST_NAME_INPUT = '[data-test="firstNameInput-text"]';
+export const CUSTOMER_INFO = '[data-test="customer-info"]';
+export const LAST_NAME_INPUT = '[data-test="lastNameInput-text"]';
+export const COMPANY_INPUT = '[data-test="companyInput-text"]';
+export const PHONE_INPUT = '[data-test="phoneInput-text"]';
+export const ADDRESS_LINE_1 = '[data-test="addressLine1Input-text"]';
+export const ADDRESS_LINE_2 = '[data-test="addressLine2Input-text"]';
+export const CITY_INPUT = '[data-test="cityInput-text"]';
+export const PROVINCE_CODE_SELECT = '[data-test="provinceCodeInput-select"]';
+export const POSTCODE_INPUT = '[data-test="postCodeInput-text"]';
+export const CHECKOUT_SHIPPING_CONTINUE = '#checkout-shipping-continue';
+export const BIGPAY_CC_NUMBER_IFRAME = '#bigpaypay-ccNumber iframe';
+export const BIGPAY_CC_EXPIRY_IFRAME = '#bigpaypay-ccExpiry iframe';
+export const BIGPAY_CC_NAME_IFRAME = '#bigpaypay-ccName iframe';
+export const BIGPAY_CC_CVV_IFRAME = '#bigpaypay-ccCvv iframe';
+export const ORDER_CONFIRMATION_HEADING = '[data-test="order-confirmation-heading"]';
+
+export * as CheckoutPage from './checkout-page';
```

Comment:
See my previous comment but we should prefer [user-facing attributes instead XPath or CSS selectors](https://playwright.dev/docs/best-practices#prefer-user-facing-attributes-to-xpath-or-css-selectors). I would just inline stuff, e.g. `await page.getByRole('textbox', { name: 'email' }).fill('example@example.com')`.